docs/cli.mdx

```diff
@@ -0,0 +1,204 @@
+
+Once supplied, this will be stored in your configuration file.
+
+:::info
+On most platforms, your config file can be found at `~/.config/svix/config.toml`.
+
+On Windows, your config file can be found at `%APPDATA%/svix/config.toml`
+:::
+
+If you would prefer not to store your AUTH_TOKEN on disk you can temporarily set it via the `SVIX_AUTH_TOKEN` env variable:
+
+ - On macOS and Linux: `export SVIX_AUTH_TOKEN=<MY-AUTH-TOKEN>`
+ - On Windows via command prompt: `set SVIX_AUTH_TOKEN=<MY-AUTH-TOKEN>`
+ - On Windows via PowerShell: `$env:SVIX_AUTH_TOKEN = '<MY-AUTH-TOKEN>'`
+
+### 3) Run your first command
+
+You’re now ready to use the Svix CLI. To get started let's create a new application:
+
+```sh
+svix application create --data-name "My Application"
+```
+
+This command creates a new application on Svix and returns the raw application object as JSON.
+
+### 4) Upgrade the Svix CLI
+
+<CLITabs>
+<TabItem value="homebrew">
+
+To upgrade the Svix CLI with [Hombrew](https://homebrew.sh), run:
+
+
+
+```sh
+brew upgrade svix/svix/svix
+```
+</TabItem>
+<TabItem value="scoop">
+
+To upgrade the Svix CLI with [Scoop](https://scoop.sh/), run:
+
+
+```sh
+scoop update svix
+```
+
+</TabItem>
+<TabItem value="macos">
+
+To upgrade the Svix CLI on macOS without Homebrew:
+
+1. Download the latest Darwin `tar.gz` file from <https://github.com/svix/svix-cli/releases/latest>
+
+2. Extract the file: `tar -xvf svix_X.X.X_Darwin_x86_64.tar.gz`
+
+2. Optionally, install the binary in a location where you can execute it globally (e.g., /usr/local/bin).
+
+</TabItem>
+<TabItem value="linux">
+
+To upgrade the Svix CLI on Linux without a package manager:
+
+
+1. Download the latest Linux `tar.gz` file from <https://github.com/svix/svix-cli/releases/latest>
+
+2. Extract the file: `tar -xvf svix_X.X.X_Linux_x86_64.tar.gz`
+
+3. Run the executable: `./svix`
+
+</TabItem>
+<TabItem value="windows">
+
+To upgrade the Svix CLI on Windows without Scoop:
+
+1. Download the latest Windows `tar.gz` file from <https://github.com/svix/svix-cli/releases/latest>
+
+2. Extract the `svix_X.X.X_Windows_x86_64.tar.gz` file.
+
+3. Run the `.exe` file
+
+</TabItem>
+</CLITabs>
+
+### 5) Get creative
+
+The Svix CLI has commands to interact with every part of the Svix API.
+
+To get a complete list of commands, run `svix --help`.
+
+Each command accepts raw JSON input as either the first positional argument or piped in over `stdin`.
+
+With this in mind, you can chain Svix CLI commands together with other command line tools like the popular JSON parser
+[jq](https://stedolan.github.io/jq/) to create powerful snippets to improve your workflow.
+
+For example, to automatically open the dashboard for the app we created in step 3, on macOS you can run the following command:
+
+
+```sh
+svix application list |  jq '.data[] | select(.name == "My Application").id' | xargs svix auth dashboard | jq '.url' | xargs open
```

Comment:
It's a bit too complex tbh, maybe change it to:
```
svix application list | jq '.data[0].id' | xargs svix auth dashboard | jq '.url' | xargs open
```

Also, there's a bug there. If the response has more than 50 apps it won't actually work.

Reply:
Yeah I know... It just felt more exciting to be able to do it by name instead of id. Updated it to be by ID as the tutorial requires app ids now anyways.